src/utils/yaml-utils.ts

```diff
@@ -1,8 +1,8 @@
 import DOMPurify from "isomorphic-dompurify"
 import yaml from "yaml"
 
```

Comment:
Nit: could we add in a comment mentioning that `yaml.parse()` or `yaml.stringify()` should never be used anywhere else in the codebase except through this function call?

Reply:
Updated in [50d2198](https://github.com/isomerpages/isomercms-backend/pull/591/commits/50d21982cffa0db19e2fc4ec9f30a71f1f690135).